packages/itstool.rb

```diff
@@ -20,9 +20,9 @@ class Itstool < Package
      x86_64: '1a2b5ed97e30e42bb37140eef1d0b09055c7f00e34b7b9748a3387fd38426f6a',
   })
 
-  depends_on 'python27'
-
-  def self.install
-    system "pip install --prefix #{CREW_PREFIX} --root #{CREW_DEST_DIR} itstool"
+  def self.build
+    system "./autogen.sh"
+    system "./configure --prefix=#{CREW_PREFIX} --libdir=#{CREW_LIB_PREFIX}"
+    system "make"
```

Comment:
Missing `def self.install` section?  Does this install any files?  What is the output of `crew files itstool`?  Also, we'll need the version bumped and the binaries removed since this is a different build.

Reply:
If I use ('the original'), the python installation, the file(poppler) can't install itself(source). But I noticed the 'possibility too', to 'do that' with /configure-make-install, the files 'are also included'.

Reply:
That's fine but we would still need a `def self.install` section, correct?

Reply:
there is a : 
def self.install
    system "pip install six"
    system "make", "DESTDIR=#{CREW_DEST_DIR}", "install"
    system "pip uninstall --yes six"
  end

Reply:
I don't see a `def self.install` section after expanding the full source:
```
require 'package'

class Itstool < Package
  description 'Translate XML with PO files using W3C Internationalization Tag Set rules'
  homepage 'http://itstool.org/'
  version '2.0.4'
  source_url 'https://github.com/itstool/itstool/archive/2.0.4.tar.gz'
  source_sha256 '3221bba908eb3c19d3b6f8e195ad706afcfef9d2f552d44e45cd77f622fd6dbc'

  binary_url ({
    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/itstool-2.0.4-chromeos-armv7l.tar.xz',
     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/itstool-2.0.4-chromeos-armv7l.tar.xz',
       i686: 'https://dl.bintray.com/chromebrew/chromebrew/itstool-2.0.4-chromeos-i686.tar.xz',
     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/itstool-2.0.4-chromeos-x86_64.tar.xz',
  })
  binary_sha256 ({
    aarch64: '65f59d6c2991f6bebd7716d11e30b443ada767f33f21c620eaafd2e8f56d3164',
     armv7l: '65f59d6c2991f6bebd7716d11e30b443ada767f33f21c620eaafd2e8f56d3164',
       i686: 'd8f925e9d45a7d758c154c59b33b942f0d9a49c51b85932aa2c6f65a6c7e00f9',
     x86_64: '1a2b5ed97e30e42bb37140eef1d0b09055c7f00e34b7b9748a3387fd38426f6a',
  })

  def self.build
    system "./autogen.sh"
    system "./configure --prefix=#{CREW_PREFIX} --libdir=#{CREW_LIB_PREFIX}"
    system "make"
  end
end
```

Reply:
Oops my bad(excuse me), I was on the 'poppler file',[that 'also needs to be changed, but I'm gonna do that tomorrow].

Otherwise 'I checked' with 'only that':
  def self.build
    system "./autogen.sh"
    system "./configure --prefix=#{CREW_PREFIX} --libdir=#{CREW_LIB_PREFIX}"
    system "make"
  end

Both Binaries and Source, work 'fine'.

Reply:
I think you might be fooled into thinking this works.  Please make sure the file is copied from the root `packages` directory to `/usr/local/lib/crew/packages` before you try to install from source.

Reply:
"make sure the file is copied from the root packages directory to /usr/local/lib/crew/packages before you try to install from source."

Yes, I use mc(midnight commander, graphical way) before each time:

[sudo mc /usr/local/lib/crew/packages /media/removable/sdcard/samske/install/executable/crew/packages]

Reply:
<a href="http://fr.tinypic.com?ref=2qjlxdd" target="_blank"><img src="http://i65.tinypic.com/2qjlxdd.png" border="0" alt="Image and video hosting by TinyPic"></a>

Reply:
I highly doubt this installs any files when installing from source.

Reply:
Yes, you're right, there is no /usr/local/bin/itstool
with the source way.

Reply:
Changed.